analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -91,6 +91,17 @@ End Sub
             return new(code, AnalyzerLanguage.VisualBasic, additionalChecks);
         }
 
+        public static SETestContext CreateVBMethod(string method, params SymbolicCheck[] additionalChecks) =>
+            new($@"
+Public Class Sample
+
+    {method}
+
+    Private Shared Sub Tag(Name As String, Optional Arg As Object = Nothing)
+    End Sub
```

Comment:
This is breaking our work with the conversions. It is better to have a generic version `Private Shared Sub Tag(Of T)(Name As String, Optional Arg As T = Nothing)` because there is no conversion happening in the call.

Reply:
This combination doesn't work, because `Tag("Name")` with optional argument cannot infer the generic type and it doesn't compile.

If we'd need it somewhere, we'll need to split those who methods to non-generic single param vs. generic with two params